### DISCUSSION TITLE
RuntimeError: shape mismatch: value tensor of shape [1616236] cannot be broadcast to indexing result of shape [1616237]

### DISCUSSION THREAD

**Discussion by zhaozzt** [open-edge-platform/anomalib#1893] [Q&A] [answered]
I have a problem here，
![9](https://github.com/openvinotoolkit/anomalib/assets/127210381/f556010f-b9e0-4014-80ba-9c35fad93c38)
Here are the details
![10](https://github.com/openvinotoolkit/anomalib/assets/127210381/0c624dda-c0d1-4b03-a518-65c25e2ec3c1)
![11](https://github.com/openvinotoolkit/anomalib/assets/127210381/b4a78e6c-2ae3-4a13-a35f-8bc291886b7d)
![12](https://github.com/openvinotoolkit/anomalib/assets/127210381/34a0fc44-6e85-4dfa-9e4d-9db9230bf48c)
![13](https://github.com/openvinotoolkit/anomalib/assets/127210381/5c9f7e4b-eae6-438a-9eff-3c3e6898ab94)
this is my data config
![14](https://github.com/openvinotoolkit/anomalib/assets/127210381/ff2ddfc4-dafd-4f98-81ae-39df71519e55)

**Comment by blaz-r** [comment]
Can you also share the model/config that you are using please.

**Reply by zhaozzt** [reply, marked as answer]
Thank you. The model I used is padim. I have solved the problem. The solution is to add the good part of the data to the test agent. I don't know if this is the reason
